frontend/src/views/Sponsor.vue

```diff
@@ -0,0 +1,204 @@
+<template>
+  <div>
+    <header id="showcase">
+      <v-img max-width="80vw" max-height="30vh" contain src="@/assets/csesocwhiteblue.png" />
+    </header>
+    <v-container class="margin" fluid>
+      <h1 class="border text-h1 font-weight-bold" style="padding:25px;">Sponsors</h1>
+      <h2 class="text-h4">Principal <br> Sponsors</h2>
+      <v-container class="border" fluid>
+          <div v-for="(sponsor, index) in tierOne" :key="sponsor.id" :style="marginStyle(index, largeLogoFilter)">
+              <div class="logo-margin">
+                <img class="large-logo" :src="sponsor.logo" v-on:click="onClickModal(sponsor)"/>
+              </div>
+              <div v-if="(index + 1) % largeLogoFilter === 0 || index === tierOne.length - 1" style="clear:both;"></div>
+          </div>
+      </v-container>
+      <div style="clear:both;"></div>
+      <h2 class="text-h4">Major <br> Sponsors</h2>
+      <v-container class="border" fluid>
+          <div v-for="(sponsor, index) in tierTwo" :key="sponsor.id" :style="marginStyle(index, midLogoFilter)">
+              <div class="logo-margin">
+                <img class="mid-logo" :src="sponsor.logo" v-on:click="onClickModal(sponsor)"/>
+              </div>
+              <div v-if="(index + 1) % midLogoFilter === 0 || index === tierTwo.length - 1" style="clear:both;"></div>
+          </div>
+      </v-container>
+      <div style="clear:both;"></div>
+      <h2 class="text-h4">Affiliate <br> Sponsors</h2>
+      <v-container class="border" fluid>
+          <div v-for="(sponsor, index) in tierThree" :key="sponsor.id" :style="marginStyle(index, smallLogoFilter)">
+              <div class="logo-margin">
+                  <img class="small-logo" :src="sponsor.logo" v-on:click="onClickModal(sponsor)"/>
+              </div>
+              <div v-if="(index + 1) % smallLogoFilter === 0 || index === tierThree.length - 1" style="clear:both;"></div>
+          </div>
+      </v-container>
+      <div style="clear:both;"></div>
+    </v-container>
+    <SponsorModal v-bind:dialog="dialog" v-bind:title="currentSponsor.name" v-bind:body="currentSponsor.detail"> </SponsorModal>
```

Comment:
The problem where modals don't show up might be here. 🤔

Reply:
Fixed this now! let me know if it works for you!

Reply:
It looks good now 😃